packages/tables/resources/views/components/button.blade.php

```diff
@@ -66,6 +67,10 @@
 @if ($tag === 'button')
     <button
         type="{{ $type }}"
+        @if($tooltip)
+        x-data="{ tooltip: '{{ $tooltip }}' }"
```

Comment:
If the `$tooltip` has any unescaped characters, it will break the rendering in the browser. We should either use `json_encode` or `@js` to properly escape the characters.

This applies to all usages of `{{ $tooltip }}` and similar.

Reply:
my apologies : I missed the last "}"